### PR TITLE
Visual design improvements

### PR DESCRIPTION
**Pull request by ricardoom**
Made some improvements to search field. mostly just minor layout stuff, margins and paddings should look more balanced when active and focused etc...

### CODE REVIEW

File: sass/_header.scss [hunk 1]
```diff
@@ -479,14 +479,14 @@ li.search {
       @include input;
       box-shadow: inset 0 0 0 2px $nypl-search-color-dark;
       height: 2rem;
-      margin-right: 1px;
+      margin: 0.3125rem 0.3125rem 0;
```
Comment (mgiraldo) [inline sass/_header.scss]: this produces a border around the search box: 
image

File: sass/_header.scss [hunk 2]
```diff
@@ -439,9 +439,9 @@ li.search {
       background-color: $nypl-search-color;
       font-size: 0.9rem;
       right: 0;
-      padding: 1px 3px 3px;
+      padding: 0.1875rem;
       position: absolute;
-      top: 2rem;
+      top: 2.65rem;
```
Comment (mgiraldo) [inline sass/_header.scss]: this moves the radio buttons far away from the box:
image

File: sass/_header.scss (hunk 1 above)
Comment (mgiraldo) [inline sass/_header.scss]: tested in Chrome, Firefox, Safari and all show it as the other screenshot 🤔

**Comment by EdwinGuzman** [comment]
I see this in Chrome.

screen shot 2017-01-11 at 2 12 56 pm

**Comment by mgiraldo** [comment]
yeah @ricardoom is working on some changes

**Comment by ricardoom** [comment]
Closing this branch & PR as it will be too difficult to reconcile the new changes to master w/ this PR

**Pull request closed without merging**